datafusion/core/src/physical_plan/rewrite.rs

```diff
@@ -149,14 +149,16 @@ pub enum RewriteRecursion {
     Skip,
 }
 
+#[allow(clippy::vtable_address_comparisons)]
```

Comment:
Do you know why  `[vtable](clippy::vtable_address_comparisons)` is needed here but not in the seemingly very similar other changes in this PR?

Reply:
Sorry, this is useless. I will remove it from the PR.